src/custom-element.js

```diff
@@ -305,13 +305,13 @@ export function createAmpElementProto(win, name, implementationClass) {
     this.isInViewport_ = false;
 
     /** @private {string|null|undefined} */
-    this.mediaQuery_;
+    this.mediaQuery_ = undefined;
 
     /** @private {!SizeList|null|undefined} */
-    this.sizeList_;
+    this.sizeList_ = undefined;
 
     /** @private {!SizeList|null|undefined} */
```

Comment:
couldnt we just assign `null` as the default then to simplify the types on these? (im ok w/ not changing these if it affects the actual code)